src/main/java/at/hannibal2/skyhanni/features/misc/items/enchants/EnchantsJson.kt

```diff
@@ -25,13 +26,18 @@ class EnchantsJson {
         return enchant
     }
 
-    fun containsEnchantment(enchants: Map<String, Int>, line: String): Boolean {
+    @Suppress("MaxLineLength")
```

Comment:
do not use this suppression. instead, expand the function parameter definion over multiple lines